src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/ProjectSystem/DocumentKey.cs

```diff
@@ -2,10 +2,10 @@
 // Licensed under the MIT license. See License.txt in the project root for license information.
 
 using System;
-using Microsoft.CodeAnalysis.Razor;
+using Microsoft.AspNetCore.Razor.ProjectSystem;
 using Microsoft.Extensions.Internal;
 
-namespace Microsoft.AspNetCore.Razor.ProjectSystem;
+namespace Microsoft.CodeAnalysis.Razor.ProjectSystem;
```

Comment:
Just starting so maybe addressed later: is there a reason the project is `Microsoft.AspNetCore.XXX` and the namespace isn't?

Reply:
What do you mean by "project"? The encloding project is "Microsoft.CodeAnalysis.Razor.Workspaces".

Reply:
Sorry, that was GH user error on my part. It turns out the yellow and blue are different 🤦 carry on

![image](https://github.com/user-attachments/assets/ed77042e-40aa-4233-936a-065b3bb3de6f)